### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by ShaheerCH**
### In this milestone
Performance was enhanced using 3NF

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+CREATE INDEX animals_id_asc ON visits (animals_id ASC);
+
+EXPLAIN ANALYZE SELECT * FROM visits where vets_id = 2;
+
+CREATE INDEX vets_id_asc ON visits (vets_id ASC);
+
+EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';
+
+CREATE INDEX email_asc ON owners (email ASC);
```
Comment (Davidosky007) [inline queries.sql]: * Kindly move all queries that modify your database( create tables and columns)  to the `schema.sql file ` as it is best practice

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @ShaheerCH @Abel-desu 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* Well written PR description 👍 
* Descriptive commit messages 👍 
* Good looking readme file 👍 

## Required Changes ♻️
* Kindly add all the screenshots with `EXPLAIN ANALYZE` results ( before and after ) since they are 3 problematic queries you should have 6 screenshots 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @ShaheerCH 🤝

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions

- [ ] The execution time in the screenshots`before` is not significantly lower than 1 sec. You might need to run this query multiple times.

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**